assets/js/graph.js

```diff
@@ -10,7 +9,7 @@ async function drawGraph(
   const container = document.getElementById('graph-container')
 
   const { index, links, content } = await fetchData
-  const curPage = url.replace(baseUrl, '')
+  const curPage = window.location.href.replace(baseUrl, '').slice(0, -1)
```

Comment:
Hm, should have a smarter way of trimming the trailing slash here. `quartz.jzhao.xyz` for example does not have a trailing slash and would break it here. Maybe do the trimming like this https://github.com/jackyzha0/quartz/commit/6e6dd4cb0b396ae7ec4f273bccd254bc0f9885d5

Reply:
That's really smart! Modified the example in 6e6dd4c a bit in 8d092a3..e646cdb